### PR TITLE
Feature/issue 140 taxonomic search

### PR DESCRIPTION
**Pull request by rasmus-storjohann-PG**

### CODE REVIEW

File: services/tests/test_api.py [hunk 1]
```diff
@@ -2,29 +2,30 @@
...
 from organizations.tests.helpers import OrganizationBuilder
+from common.testhelpers.autofixtures import *
 
 class ServicesApiTests(rest_test.APITestCase):
     def setUp(self):
-        self.organization_id = 'the_organization_id'
+        self.organization_id = a_string()
         self.organization = OrganizationBuilder().with_id(self.organization_id).build()
         self.organization.save()
         self.data = {
```
Comment (bensinjin) [inline services/tests/test_api.py]: These data values should be being used in the other tests in this file instead of re calling a_string().

Comment (bensinjin) [inline services/tests/test_api.py]: These data values should be being used in the other tests in this file instead of re calling a_string().

Comment (bensinjin) [inline services/tests/test_api.py]: These data values should be being used in the other tests in this file instead of re calling a_string().

Comment (bensinjin) [inline services/tests/test_api.py]: These values should be used throughout ServicesApiTests as apposed to re calling a_string().

Comment (rasmus-storjohann-PG) [reply services/tests/test_api.py]: self.data is used for post and put calls only.

**Pull request merged**